src/client/containers/App.tsx

```diff
@@ -45,7 +49,9 @@ export const App: React.FC = () => {
     )
   }
 
-  return (
+  return width <= 760 ? (
+    <MobileMsg></MobileMsg>
```

Comment:
I think this can be shortened to `<MobileMsg />`

Reply:
Done.